src/Chapter1.hs

```diff
@@ -551,7 +565,8 @@ True
 >>> isVowel 'x'
 False
 -}
-isVowel c = error "isVowel: not implemented!"
+isVowel :: Char -> Bool
+isVowel c = elem c "aeiouyAEIOUY"
```

Comment:
Absolutely correct! And you use eta-reduction 👍🏼 

One note in here, that sometimes, `elem` could be slower than the explicit pattern matching. I remember there were some benchmarks on one particular case, that showed how moving to pattern matching on each case separately drastically decrease time 🐎

Reply:
That's a bit surprising. Intuitively, I would expect a built in common function like `elem` to have some optimizations compared to the naive approach.

Reply:
Yes, `elem` has optimizations, but this particular case wasn't covered by optimizations. It was discovered accidentally only recently. But, fortunately, this should be fixed in the next GHC version :slightly_smiling_face: